builtin/config.c

```diff
@@ -189,11 +191,60 @@ static void show_config_origin(struct strbuf *buf)
 	strbuf_addch(buf, term);
 }
 
+static const char *scope_to_string(enum config_scope scope) {
+	const char *local = "(local)";
```

Comment:
Should these be translatable? If so, please surround them in `N_(...)`, i.e.

```c
const char *local = N_("local");
```

Reply:
the prefixes for `--show-origin` were not translatable so I didn't consider it, but I don't see any reason why these shouldn't be